ledger/jetcoordinator/jetnode.go

```diff
@@ -0,0 +1,28 @@
+package jetcoordinator
+
+import (
+	"bytes"
+
+	"github.com/insolar/insolar/core"
+)
+
+type JetNode struct {
+	ref   core.RecordRef
+	left  *JetNode
+	right *JetNode
+}
+
+func (jn *JetNode) GetContaining(objRef *core.RecordRef) *core.RecordRef {
+	if jn.left == nil || jn.right == nil {
+		return &jn.ref
+	}
+
+	switch bytes.Compare(objRef[:], jn.ref[:]) {
+	case 0:
+		return &jn.ref
+	case -1:
+		return jn.left.GetContaining(objRef)
+	default:
+		return jn.right.GetContaining(objRef)
+	}
```

Comment:
Why 3 cases? Only leafs of tree should represent Jets

Reply:
case 0: its jet itself
The rest two are for branches.